tasks/main.yml

```diff
@@ -11,4 +11,6 @@
   - include_tasks: elasticsearch_search_config.yml
   - include_tasks: elasticsearch_post.yml
   tags:
-    - elasticsearch-config
\ No newline at end of file
+    - elasticsearch-config
+
+- include_tasks: elasticsearch_activate.yml
```

Comment:
You can put this below include_tasks for faster run

```
   when: elasticsearch_license_check.stat.exists 
```

Reply:
Ok, Done krub